staff/alicia-alcaide/array-functions/concat.js

```diff
@@ -0,0 +1,33 @@
+/**
+ *  Used to merge two or more arrays. 
+ *  This method does not change the existing arrays, but instead returns a new array.
+ * 
+ * @param {*} values Arrays and/or values to concatenate into a new array. If valueN is undefined, concat returns a shallow copy of the existing array on which it is called. See the description below for more details.
+ * 
+ * @returns {Array} A new Array instance.
+ */
+
+'use strict';
+
+function concat() {
+
+
+  var newArray = [];
+  
+
+
+  for (var i=0; i<arguments[0].length; i++) {
+
+    newArray[i] = arguments[0][i];
+    
+  }
+  
+  for (var j=1; j <arguments.length; j++) {
+    for (var k=0; k < arguments[j].length; k++) {
+        newArray[i] = arguments[j][k];
+        i++;
```

Comment:
¿i? ¿qué es i?

Reply:
i es el índice del nuevo array, venia declarado y con el valor inicial que le tocaba del for anterior. Al quitar dicho for lo declaro e inicializo antes de este for. Son nesesarios tres índices: este (i) del nuevo array, j que itera sobre arguments para y por último k que itera dentro de cada uno de los arrays dentro de arguments.

Reply:
¿Y por qué no haces newArray[newArray.length]? creo que se entendería mejor

Reply:
modificado